### PR TITLE
common: improve documentation comments

### PR DESCRIPTION
**Pull request by kielbarry**

### CODE REVIEW

File: common/bytes.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package common
 
 import "encoding/hex"
 
+// ToHex returns string representation of b and either prepends '0x' or initializes value of '0'.
```
Comment (ligi) [inline common/bytes.go]: this is wrong - it always prepends 0x

File: common/bytes.go [hunk 2]
```diff
@@ -28,6 +29,7 @@ func ToHex(b []byte) string {
 	return "0x" + hex
 }
 
+//FromHex returns bytes of s after removing prefix, or prepends 0 if s has odd length.
```
Comment (ligi) [inline common/bytes.go]: I think this comment is confusing more than it helps

Comment (kielbarry) [reply common/bytes.go]: Hey @ligi, Apologies for my mistakes, and thank you for your feedback!

File: common/bytes.go (hunk 1 above)
Comment (kielbarry) [reply common/bytes.go]: Apologies, thank you for your fix

File: common/types.go [hunk 3]
```diff
@@ -117,6 +119,7 @@ func (h Hash) Generate(rand *rand.Rand, size int) reflect.Value {
 	return reflect.ValueOf(h)
 }
 
+// EmptyHash checks whether h is the zero hash.
 func EmptyHash(h Hash) bool {
 	return h == Hash{}
 }
```
Comment (karalabe) [inline common/types.go]: Please delete this method, there's really no point to have this method (also it's a pointless function call overhead).

File: common/path.go [hunk 4]
```diff
@@ -39,6 +40,7 @@ func FileExist(filePath string) bool {
 	return true
 }
 
+// AbsolutePath returns Datadir + filename, or filename if it is absolute.
 func AbsolutePath(Datadir string, filename string) string {
```
Comment (karalabe) [inline common/path.go]: Please lowercase `Datadir` here, not sure how it got capitalized in the first place.

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 75fa19456962a8db1a9cb0564a2c9be96e47a428
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by fjl** [comment]
Build is fixed. PTAL.

**Comment by kielbarry** [comment]
@fjl I just pulled and tests/builds are passing locally for me.

**Pull request merged**